clients/ui/frontend/package.json

```
@@ -16,9 +16,19 @@
     "build:bundle-analyze": "webpack-bundle-analyzer ./bundle.stats.json",
     "build:clean": "rimraf ./public",
     "build:prod": "webpack --config ./config/webpack.prod.js",
-    "start:dev": "webpack serve --hot --color --config ./config/webpack.dev.js"
+    "start:dev": "webpack serve --hot --color --config ./config/webpack.dev.js",
+    "test:cypress-ci": "npx concurrently -P -k -s first \"npm run cypress:server:build && npm run cypress:server\" \"npx wait-on tcp:127.0.0.1:9001 && npm run cypress:run:mock -- {@}\" -- ",
+    "test:jest": "jest",
+    "test:watch": "jest --watch",
+    "test:coverage": "jest --coverage",
+    "cypress:run": "cypress run -b chrome --project src/__tests__/cypress",
+    "cypress:run:mock": "CY_MOCK=1 npm run cypress:run -- ",
```

```suggestion
    "cypress:run:mock": "CY_MOCK=1 npm run cypress:run -- ",
     "cypress:run": "cypress run -b chrome --project src/__tests__/cypress",
    "cypress:open": "cypress open --project src/__tests__/cypress",
```

Can we add `cypress:open` too, to have the cypress dashboard? It's quite handy and works really great to debut stuff, if you like that, we can add it to the testing docs too.

---

In the dashboard docs:

```
There are two commands to run Cypress mock tests (always use the `:mock` variants).
- `open`: Open the Cypress GUI
  \`\`\`bash
  npm run cypress:open:mock
  \`\`\`
- `run`: Run all Cypress tests or a specific test headless
  \`\`\`bash
  npm run cypress:run:mock

  npm run cypress:run:mock -- --spec "**/testfile.cy.ts"
  \`\`\`
```